ksqldb-engine/src/main/java/io/confluent/ksql/analyzer/Analyzer.java

```diff
@@ -169,14 +169,18 @@ private void analyzeNonStdOutSink(final Sink sink) {
           .getDataSource()
           .getKsqlTopic();
 
-      final FormatInfo keyFmtInfo = buildKeyFormatInfo(
```

Comment:
Are the changes in this file purely a refactor to have separate helpers for getting format name and info, or are there functional differences here?

Reply:
yes, just a refactor because it made it easier to share the `buildFormatInfo` method across the key/value